plugins/modules/idrac_server_config_profile.py

```diff
@@ -798,6 +852,69 @@ def get_buffer_text(module, share):
     return buffer_text
 
 
+def get_xml_content(module, xml_content):
+    idrac_content = None
+    try:
+        root = ET.fromstring(xml_content)
+        component = next((c for c in root.iter('Component') if c.get('FQDD') == 'iDRAC.Embedded.1'), None)
```

Comment:
Can we avoid hardcoding based on FQDD, if not can you have this in the variable above?